src/test/java/com/fasterxml/jackson/core/json/StringGenerationTest.java

```diff
@@ -86,6 +87,17 @@ void longerRandomMultiChunk() throws Exception
         }
     }
 
+    @Test
+    public void testWritingSurrogatePairs() throws IOException {
```

Comment:
Note: there is already

src/test/java/com/fasterxml/jackson/failing/Surrogate223Test.java

that hopefully now passes; if so, should be moved to another location from under `.../failing`

Reply:
Confirmed. Surrogate223Test is green now.